patches/0028-core-Disable-long-running-tests.patch

```diff
@@ -0,0 +1,30 @@
+From 0ffdd8eda8df364391c8ac6e1ce92c73ba9254d4 Mon Sep 17 00:00:00 2001
+From: bjorn3 <bjorn3@users.noreply.github.com>
+Date: Fri, 3 Dec 2021 12:16:30 +0100
+Subject: [PATCH] Disable long running tests
+
+---
+ library/core/tests/slice.rs | 3 +++
+ 1 file changed, 3 insertions(+)
+
+diff --git a/library/core/tests/slice.rs b/library/core/tests/slice.rs
+index 2c8f00a..44847ee 100644
+--- a/library/core/tests/slice.rs
++++ b/library/core/tests/slice.rs
+@@ -2332,7 +2332,8 @@ macro_rules! empty_max_mut {
+     };
+ }
+ 
++/*
```

Comment:
Should this be enabled back?
Not sure why this was disabled.

Reply:
These tests only finish within reasonable time when optimizations are enabled. The libcore tests are currently tested with optimizations disabled.

Reply:
Oh wow, I did not notice that. I guess those are new tests. Let's keep them disabled, then.